assets/models/system/codellama-13b-instruct-hf/spec.yaml

```diff
@@ -3,15 +3,15 @@ name: CodeLlama-13b-Instruct-hf
 path: ./
 properties:
   datasets: Publicly available sources
-  finetuning-tasks: text-classification
-  inference-recommended-sku: Standard_NC24s_v3, Standard_ND40rs_v2, Standard_ND96asr_v4,
-    Standard_ND96amsr_A100_v4
+  inference-recommended-sku: Standard_NC24s_v3
+  evaluation-recommended-sku: Standard_NC24s_v3
   languages: EN
   azureml.copyRegistryFilesToWorkspace: true
 tags:
   Featured: ''
   Preview: ''
   computes_allow_list:
+  - Standard_NC24s_v3
```

Comment:
remove compute_allowlist and have scenario specific allowlist

Reply:
updated